src/main/java/k2b/K2B.java

```diff
@@ -0,0 +1,87 @@
+package k2b;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class K2B {
+
+    private Map<Character, Character> kbMap;
+
+    public K2B() {
+        this.initKBMap();
+    }
+
+    private void initKBMap() {
```

Comment:
Take a look about this http://stackoverflow.com/questions/507602/how-can-i-initialize-a-static-map